.github/workflows/appimage.yml

```diff
@@ -26,7 +34,7 @@ jobs:
     timeout-minutes: 30
 
     steps:
-    - name: Check out main code
```

Comment:
Why remove 'main'? (We still have "Check out PR branch code" below)

Reply:
I just saw that was how RDK was doing it but can revert if desired

Reply:
Sounds good if this is what we do in RDK